stable/prometheus-operator/Chart.yaml

```diff
@@ -11,7 +11,7 @@ name: prometheus-operator
 sources:
   - https://github.com/coreos/prometheus-operator
   - https://coreos.com/operators/prometheus
-version: 5.14.1
+version: 5.14.2
```

Comment:
Since this is a new feature, rather than a bugfix, you should be bumping the minor version, rather than the patch.
https://semver.org/

Reply:
Ok. Updated.